### PR TITLE
Add workshops

### PR DESCRIPTION
**Pull request by yaigrie**

### CODE REVIEW

**Review by marcoow — COMMENTED** [review]
I don't think this is quite there yet design-wise – should we first figure out a design in Sketch?

**Comment by yaigrie** [comment]
@marcoow Sure thing! Do we only need the single workshop pages or anything else?

File: templates/sections/schedule-temp.html [hunk 1]
```diff
@@ -2,10 +2,6 @@
   <div class="container">
     <h5 class="mb-4">Schedule</h5>
     <h2 class="mb-7">This is your EuroRust experience</h2>
-    <div class="workshops mb-9">
-      <h4 class="mb-4">Workshop Day - Oct. 9th, 2024</h4>
-      <p>On the day before the main conference, we'll be running workshops again. Details about the workshops will be announced soon so stay tuned and make sure to plan your travel accordingly!</p>
-    </div>
```
Comment (marcoow) [inline templates/sections/schedule-temp.html:8]: Let's keep this section and just remove the _"Details about the…"_

File: templates/sections/workshop-schedule.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<section class="schedule border-t py-10">
+  <div class="container">
+    <h5 class="mb-4">Workshops</h5>
+    <h2 class="mb-4">Experience Rust in action</h2>
```
Comment (marcoow) [inline templates/sections/workshop-schedule.html:4]:
```suggestion
    <h2 class="mb-4">Deepen your Rust skills hands-on!</h2>
```

File: templates/sections/workshop-venue.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+			<div class="venue__wrapper-image">
+				<div class="venue__wrapper-image-overlay"></div>
+				<img class="image-venue" src="/images/venue/venue.jpg" alt="venue picture"/>
+			</div>
+			<div class="venue__wrapper-content">
+				<h5 class="mb-4">Venue</h5>
+				<h2 class="mb-4">The place to be</h2>
+				<p class="mb-8">EuroRust will take place in Vienna, in the WIENER WERKSHALLEN. The venue is located at 2. Haidequerstraße 1-3, 1110 Vienna, Austria.</p>
+				<a class="btn__secondary" href="https://maps.app.goo.gl/XkMsdSps3PQibanP8" alt="Google Maps for conference venue address"
+				target="_blank" rel="noopener noreferrer">Check on Map</a>
```
Comment (marcoow) [inline templates/sections/workshop-venue.html:13]: The workshops aren't in the conference venue but the workshop is tba. This needs to repeat the text from the main landing page: "The EuroRust workshops will take place in a separate venue…" with a placeholder image

**Review by marcoow — CHANGES_REQUESTED** [review]
Should the mentor image be square like the speaker images on the main landing page? 
Bildschirmfoto 2024-04-26 um 15 02 30

File: templates/sections/workshop-schedule.html (hunk 2 above)
Comment (marcoow) [reply templates/sections/workshop-schedule.html:4]: @yaigrie let's still do this as well please

**Review by marcoow — COMMENTED** [review]
Let's still make the mentor images square – I think `aspect-ratio: 1/1;` only works in a few browsers

**Comment by marcoow** [comment]
@yaigrie https://github.com/mainmatter/eurorust.eu/pull/454#discussion_r1581134096 🙏

File: content/workshops/async-rust.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
++++
+title = "Rediscovering the Future: a deep dive into the development of async Rust"
```
Comment (LukeMathWalker) [inline content/workshops/async-rust.md:2]:
```suggestion
title = "Rediscovering the Future: a deep dive into async Rust"
```

I think it works better if it's a bit shorter.

**Review by marcoow — APPROVED** [review]

**Pull request merged**